skll/config.py

```diff
@@ -338,6 +353,17 @@ def _parse_config_file(config_path):
     custom_learner_path = _locate_file(config.get("Input", "custom_learner_path"),
                                        config_dir)
 
+    # Check that the learners are valid learner types (or that they are custom
+    # learners, i.e., ending in ".py")
```

Comment:
I have removed the check. I think I was just adding in a check because I didn't know it was done elsewhere. It might be nice to have a check here so that mistakes get recognized immediately, but I don't have a strong opinion about this.

However, before I removed it, I just want to mention that I did try the other method you provided an example of to get the unrecognized learner types. I couldn't get it to work correctly. `globals()` does not seem to include anything of the form `skll.learner.x` where `x` is the learner name. Nor does it include just the learner name itself. I tried making modifications to it, but I could not get this to work. I mention this here mainly because it will tie into what I have to say later about using `globals()` to get the classes that correspond to learner names.